### PR TITLE
initial support for nodejs8

### PR DESCRIPTION
**Pull request by csantanapr**
- [x] Add gradle project for nodejs8
- [x] Add tests for nodejs8
- [x] Update docs for nodejs8
- [x] Update swagger definition with new kind

### CODE REVIEW

File: actionRuntimes/nodejs8Action/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM csantanapr/action-nodejs-v8
```
Comment (csantanapr) [inline actionRuntimes/nodejs8Action/Dockerfile]: TODO: update to `openwhisk/action-nodejs-v8:1.0.0`

**Comment by csantanapr** [comment]
TODO: update docs

**Comment by csantanapr** [comment]
PG1 2310 🔵

**Comment by csantanapr** [comment]
@rabbah bump

File: docs/actions.md [hunk 2]
```diff
@@ -493,6 +494,90 @@ To create an OpenWhisk action from this package:
...
+
+3. Create the Action using the `npm` script or the CLI.
+  Using `npm` script:
+  ```
+  npm run deploy
+  ```
+  {: pre}
+  Using the CLI:
+  ```
+  bx wsk action update my-action dist/bundle.js
```
Comment (rabbah) [inline docs/actions.md]: what's bx?

Comment (csantanapr) [reply docs/actions.md]: sorry about that will fix

**Comment by rabbah** [comment]
one minor issues otherwise LGTM.

**Comment by csantanapr** [comment]
fixed now

**Comment by csantanapr** [comment]
@rabbah doc fixed

**Pull request merged**